4788/src/main/include/Robot.h

```diff
@@ -18,8 +49,15 @@ class Robot : public frc::TimedRobot, protected wml::StrategyController, protect
 	void TestInit() override;
 	void TestPeriodic() override;
 
- private:
-	// Istances
-	RobotMap robotMap;
-	wml::Drivetrain *drivetrain; // WML drivetrain (Not local)
+	//controllers
+	frc::XboxController *xbox;
+
+	//motors
+	frc::Spark *SparkMotor[1];
```

Comment:
You'll probably be fine just leaving this line as `frc::Spark *SparkMotor` rather than `frc::SparkMotor[1]`